pkgs/development/python-modules/pywebcopy/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  python3Packages,
+  pytestCheckHook,
```

Comment:
You should add all dependencies such as setuptools,six,request etc here

Reply:
Why doing this if the building still works with no adding those deps that are retrieved from `python3Packages`?

Reply:
python3Packages always points to python312Packages but if you build python311Packages.pywebcopy, you would mix python versions which breaks things.